pkg/operator/operator.go

```diff
@@ -276,6 +277,18 @@ func New(
 	})
 	o.informers = append(o.informers, informer)
 
+	informer = cache.NewSharedIndexInformer(
```

Comment:
Do we need the watch and informer infra for this use case?

Reply:
This informer is for the case where the console initialize its route, and trigger a change in the custom resource "console". So CMO can update immediately the externalURL for AlertManager using the URL to console.

Reply:
Does the informer consume many resources and we may not need the externalURL of AlertManager set immediately when the console becomes available?

Reply:
Using an informer, the operator doesn't have to wait for the console operator to complete. I would expect that we watch only the cluster resource though like we do for [infrastructure config](https://github.com/openshift/cluster-monitoring-operator/blob/320ea1b465d8aa398ae1aefa05e67dc048fa33dc/pkg/operator/operator.go#L258-L261).